iocore/net/UnixNetAccept.cc

```diff
@@ -31,6 +31,7 @@
 using NetAcceptHandler = int (NetAccept::*)(int, void *);
 int accept_till_done   = 1;
 
+Ptr<ProxyMutex> naVecMutex;
```

Comment:
Isn't this a different mutex than the one in `SSLSNIConfig.cc`?

Reply:
it's the same one